src/ray/raylet/worker_pool.cc

```diff
@@ -16,27 +16,46 @@ WorkerPool::WorkerPool(int num_workers, const std::vector<std::string> &worker_c
   // become zombies instead of dying gracefully.
   signal(SIGCHLD, SIG_IGN);
   for (int i = 0; i < num_workers; i++) {
-    StartWorker();
+    // Force-start num_workers workers.
+    StartWorker(true);
```

Comment:
I learned this tip from Zongheng to write this as `StartWorker(/*force_start*/=true)` to make it clear to the reader what the parameter is. :)

Reply:
doesn't this introduce C-style comments? I'd rather not, if there's a cleaner, pythonic way, that'd be great.